.eslintrc

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
That's ok, I'll defer :stuck_out_tongue_closed_eyes: 

As long as the spacing/line breaks stay, I'm happy. It was so hard to follow the function chaining & parentheses wrapping before. Basically relied on my editor to track & highlight the pairs every time.